iphone/Classes/TiUIiOSProxy.h

```diff
@@ -171,6 +171,11 @@
 @property(nonatomic,readonly) NSNumber* BLUR_EFFECT_STYLE_DARK;
 #endif
 
+#if IS_XCODE_9
+@property(nonatomic,readonly) NSNumber* LARGE_TITLE_DISPLAY_MODE_AUTOMATIC;
```

Comment:
Lets discuss about it offline. There are already similar properties mentioned in .h file. If without writing the same  it is working fine, then no need to write extra line of code. Every property which we make using MAKE_SYSTEM_PROP or MAKE_SYSTEM_STR etc, are available to user.